### PR TITLE
NN-3506:

### PR DESCRIPTION
**Pull request by JamieDraperUK**
- Library upgrades

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.3.0"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.3.3"
   kotlin("plugin.spring") version "1.5.10"
```
Comment (petergphillips) [inline build.gradle.kts:5]: needs bump of kotlin for 3.3.3

Comment (petergphillips) [reply build.gradle.kts:5]: should read
```
  kotlin("plugin.spring") version "1.5.20"
```

File: build.gradle.kts [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   kotlin("plugin.jpa") version "1.5.10"
```
Comment (petergphillips) [inline build.gradle.kts:6]: should read
```
  kotlin("plugin.jpa") version "1.5.20"
```

File: build.gradle.kts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```
Comment (andrewrlee) [inline build.gradle.kts:1]: is it worth bumping the orb to add the change log printing?

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.5
+  hmpps: ministryofjustice/hmpps@3.7.0
```
Comment (petergphillips) [inline .circleci/config.yml:4]: should be 3.8, don't need to pin to patch version

Comment (petergphillips) [reply .circleci/config.yml:4]: but need 3.8 to get veracode fix

File: .circleci/config.yml [hunk 5]
```diff
@@ -78,6 +78,7 @@ workflows:
           type: approval
           requires:
             - deploy_preprod
+          show_changelog: true
```
Comment (petergphillips) [inline .circleci/config.yml:81]: don't need this line - default from 3.6 onwards is to show the changelog

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**